lib/utilities/monitoringHandler.js

```diff
@@ -1,6 +1,107 @@
 const errors = require('arsenal');
 const client = require('prom-client');
 
+const collectDefaultMetrics = client.collectDefaultMetrics;
+let crrStatsPulled = false;
+const numberOfBuckets = new client.Gauge({
+    name: 'cloud_server_number_of_buckets',
+    help: 'Total number of buckets',
+});
+const numberOfObjects = new client.Gauge({
+    name: 'cloud_server_number_of_objects',
+    help: 'Total number of objects',
+});
+const dataDiskAvailable = new client.Gauge({
+    name: 'cloud_server_data_disk_available',
+    help: 'Available data disk storage in bytes',
+});
+const dataDiskFree = new client.Gauge({
+    name: 'cloud_server_data_disk_free',
+    help: 'Free data disk storage in bytes',
+});
+const dataDiskTotal = new client.Gauge({
+    name: 'cloud_server_data_disk_total',
+    help: 'Total data disk storage in bytes',
+});
+
+const labelNames = ['method', 'service', 'code'];
+const httpRequestsTotal = new client.Counter({
+    labelNames,
+    name: 'cloud_server_http_requests_total',
+    help: 'Total number of HTTP requests',
+});
+
+const httpRequestSizeBytes = new client.Summary({
+    labelNames,
+    name: 'cloud_server_http_request_size_bytes',
+    help: 'The HTTP request sizes in bytes.',
+});
+
+const httpResponseSizeBytes = new client.Summary({
+    labelNames,
+    name: 'cloud_server_http_response_size_bytes',
+    help: 'The HTTP response sizes in bytes.',
+});
+
+function promMetrics(requestType, bucketName, code, typeOfRequest,
+    newByteLength, oldByteLength, isVersionedObj,
+    numOfObjectsRemoved) {
+    httpRequestsTotal.labels(requestType, 'cloud_server', code).inc();
+    if ((typeOfRequest === 'putObject' ||
+    typeOfRequest === 'copyObject' ||
+    typeOfRequest === 'putOjectPart') &&
+    code === '200') {
+        httpRequestSizeBytes
+            .labels(requestType, 'cloud_server', code)
+            .observe(newByteLength - (isVersionedObj ? 0 : oldByteLength));
```

Comment:
`newByteLength - (isVersionedObj ? 0 : oldByteLength)` This can be a var as you are re-using it in multiple places.

Reply:
made the change